Mathlib/Tactic/Congr!.lean

```diff
@@ -46,7 +46,7 @@ structure Congr!.Config where
   By default this is `.reducible`, which prevents unfolding of most definitions. -/
   preTransparency : TransparencyMode := TransparencyMode.reducible
   /-- For passes that synthesize a congruence lemma using one side of the equality,
-  we run the pass both for the left-hand side and the right-hand side. If `preferLHS` is `true`
+  we run the pass both for the left-hand side and the right-hand side. If `preferLHS` is `True`
```

Comment:
`Bool.true` is exported to the root namespace, so `true` is the Lean 4 name for the value of `Bool`, so this is a bad fix (also elsewhere). `true`/`false` should only be capitalized if the type is meant.

Reply:
Now this makes sense, thank you! I have hopefully fixed all instances.